src/observers/signStatusBarObserver.ts

```diff
@@ -28,7 +28,6 @@ export class SignStatusBarObserver extends BaseStatusBarObserver {
                 let asCredentialRetrieveFromLocalCredentialManager = <CredentialRetrieveFromLocalCredentialManager>event;
                 this.handleSignedIn(asCredentialRetrieveFromLocalCredentialManager.credential);
                 break;
-            case EventType.UserSignedOut:

```

Comment:
no need to refresh the status bar for `UserSignedOut`?

Reply:
when the user sign out, the `resetCredential` function will be called, and a `CredentialReset` event will be post.